src/changes.apib

```diff
@@ -48,6 +48,26 @@ _This will only affect the version for those API calls and won't affect any othe
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and will not appear in this list.
 
+#### 2019-01-24
+
+- Previously, the `invoices.info` endpoint returned an _non-existing_ or _empty_ `extended_description` as an empty string, while `invoices.draft` and `invoices.update` didn't allow an empty string as `extended_description`.  
+  From now on, `invoices.info` will return an _non-existing_ or _empty_ `extended_description` as `null`, to make this more consistent and interchangeable.
+
+- Invoices can now have three statuses (`draft`, `outstanding` or `matched`) instead of two (`draft` or `booked`).  
+  Therefore the `status` property in `invoices.list` and `invoices.info` will either return `outstanding` or `matched` for invoices that were previously just `booked`.  
+  This is also reflected in the `status` filter on `invoices.list`: to allow you to still query all `outstanding` and `matched` invoices (previously all `booked`) , it now *only* accepts an array of filter values (eg. `["outstanding", "matched"]` or `["draft"]`).
+
+- The property `local_business_number` was renamed to `national_identification_number` on the following endpoints:  
+  `companies.info`,  
+  `companies.list`,  
+  `companies.add`,  
+  `companies.update`,  
+  `invoices.info`  
+  and `creditNotes.info`.
```

Comment:
B: this is really showing on distinct lines in the generated HTML. Can we put these on a single line?

Reply:
seems rather PP to me 🤔

Reply:
tbh, I actually want readers to spot endpoint names easily in the changelogs, so I would like to try it like this...

Reply:
Can we use list items in that case? Because it looks like faulty HTML or an unintended result right now.